features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/blocks/application/application-update/ajax/application-update.jag

```diff
@@ -52,6 +52,11 @@ include("/jagg/jagg.jag");
             callbackUrlNew = request.getParameter("callbackUrlNew");
         }        
         descriptionNew = request.getParameter("descriptionNew");
+        if (request.getParameter("applicationAttributeNew") == null) {
```

Comment:
Same is https://github.com/wso2/carbon-apimgt/pull/5392/files#r271594297

Reply:
Fixed in #6098